pkg/api/const.go

```diff
@@ -252,6 +252,9 @@ const (
 
 	// DefaultAzureStackFaultDomainCount set to 3 as Azure Stack today has minimum 4 node deployment.
 	DefaultAzureStackFaultDomainCount = 3
+
+	// DefaultAzureStackEtcdDiskSizeGT10Nodes = size for Kubernetes master etcd disk volumes in GB if > 10 nodes
+	DefaultAzureStackEtcdDiskSizeGT10Nodes = "1023"
```

Comment:
the naming "GT10Nodes" doesn't really apply here since there's only one default size

Reply:
My Bad. Updating name to MaxAzureStackManagedDiskSize to be consistent to another PR.